tests/onnxruntime/nightly_test_trainer.py

```diff
@@ -674,6 +676,37 @@ def test_trainer_fp16_ds_stage2(self, test_name, model_name, feature, data_metri
             gc.collect()
 
 
+# @slow
+# @require_torc
+class ORTTrainerIntegrationDDPTest(unittest.TestCase):
+    def test_trainer_ddp_glue(self):
+
+        subprocess.run(
+            f"cp examples/onnxruntime/training/text-classification/run_glue.py ./",
+            shell=True,
+        )
+
+        subprocess.run(
+            f"{sys.executable} -m torch.distributed.launch"
+            " --nproc_per_node=1"
```

Comment:
Could we test it with more than one device?

Reply:
If not mistaken, the current ec2 instance in CI only has one GPU. Do you test with multiple for Habana?

Reply:
@regisss, let's leave it to one process per node and increase the GPU number for CI in another PR if necessary.

Reply:
Ah okay I thought the type of instance you use had more than one device! No worries it looks good to me :+1: